benchmark/ios/BenchmarkApp_iphoneos13.2-arm64e-release.xctestrun

```diff
@@ -15,8 +15,8 @@
 		<array/>
 		<key>DependentProductPaths</key>
 		<array>
-			<string>__TESTROOT__/Release-iphoneos/BenchmarkApp.app</string>
-			<string>__TESTROOT__/Release-iphoneos/BenchmarkApp.app/PlugIns/BenchmarkAppTests.xctest</string>
+			<string>__TESTROOT__/Release/BenchmarkApp.app</string>
+			<string>__TESTROOT__/Release/BenchmarkApp.app/PlugIns/BenchmarkAppTests.xctest</string>
```

Comment:
Can we add the `SDKROOT` back to the build products path? As it is, I think cross-platform targets like mbgl-core end up in generic folders like build/Release/ and build/Debug/, which means the iOS and macOS builds overwrite each other. Simulator and device builds may also override each other when these paths collide.